assessment.md

```diff
@@ -119,56 +119,57 @@ when creating python programs.
   * E = Issue is unique, well described, and a possible reason for occurrence
     and solution is mentioned
 * Using the GitHub flow model
-  * N = GitHub flow model unused or no commits
-  * I = Pushed to master or non-descriptive commits
-  * A = Pushed to correct repository with descriptive commits
+  * N = GitHub flow model unused or no commits to the Github repository
+  * I = Pushed to master or non-descriptive commits to the Github repository
+  * A = Pushed to correct repository with descriptive commits to the Github
+  repository
   * G = Use of a branch/fork demonstrated appropriately
   (e.g having a feature branch) along with descriptive commits
   * E = Use of multiple useful branches/forks demonstrated appropriately
-    containing multiple coherent and descriptive commits
+    containing multiple consistent and descriptive commits
```

Comment:
Coherent and consistent do not mean the same thing -> is the change in meaning acceptable?

I would argue that coherent commits are incredibly necessary.

Reply:
I no longer have the paper with me, but if I remember correctly this change was suggested by @gkapfham.